src/org/thoughtcrime/securesms/util/Stereotype.java

```diff
@@ -26,7 +26,7 @@ public static String fromBody(String body) {
 
         switch(type) {
             case REACTION:
-            case REPLY:
+            case COMMENT:
```

Comment:
@g-harel can you add the tests for comment type as well ?? I can send the you exact json of a comment obj if you want